lib/netflix2.js

```diff
@@ -205,6 +205,59 @@ Netflix.prototype.__getViewingHistory = function (page, callback) {
   })
 }
 
+/**
+ * Hides viewing history for a specific movie or series
+ * @param movieID  - the ID of the movie (e.g. 80057281 for "Stranger Things")
+ * @param seriesAll - true if you want to clear the whole series, false otherwise
+ */
+Netflix.prototype.hideMovieViewingHistory = function (movieID, seriesAll, callback) {
```

Comment:
Two things about this function's name:

1. It can also hide series from the viewing history and I think that sould be reflected in the function's name. Also, I would add a "from" to not make it sound like you're hiding something called a "movie viewing history".
2. A call to this function would look something like
```javascript
netflix.hideMovieViewingHistory(someMovieId, true, someFunction);
```
The `true` would seem completely random and the user would have to look up the source code or the documentation to understand its purpose. I would suggest either

a) passing something like a config object to the function:
```javascript
netflix.hideMovieViewingHistory({ movieId: someMovieId, hideEntireSeries: true }, someFunction);
```
or b) splitting the function up into two separate functions (for example `hideSingleMovieFromViewingHistory` and `hideEntireSeriesFromViewingHistory`) to clarify their functionality.

Please make sure to combine both suggestions (as long as you don't disagree with either of them, of course).

Also: Are there multiple movies that combine to a series? For example: If I were to set `seriesAll = true` when hiding one of the Fast and Furious movies, would I then hide all of them or does that only work for actual series? If it is the latter, that might be worth mentioning either in the documentation or even in the function / parameter names as well.

Reply:
Interesting suggestions. I tried to follow `getViewingHistory`. 

I would also like to suggest using `Activity` instead of `History` in the name to make the function easier to be recognised in relation to the actual request being made (`https://www.netflix.com/api/shakti/vb6c61687/viewingactivity`)

For example:
```javascript
netflix.hideViewingActivity({ movieID: someMovieId, seriesAll: true }, someFunction);
```

I agree passing a config object is more readable: don't need to think what the arguments are and what they mean. I was unsure how consistently that is used throughout the API.
For example login uses that, but the majority of the other method seem to use arguments, not a config object.

Would something like this work ?

```javascript
netflix.hideMovieViewingActivity(movieID, someFunction);
netflix.hideSeriesViewingActivity(movieID, someFunction);
```

Reply:
The API's consistency is a really good point! However, I think that `hideSeriesViewingActivity` and `hideMovieViewingActivity` would imply that one of the functions is for movies and that the other is for series (or shows). This might be because of my German background, but even if your suggestion is absolutely correct, we should nevertheless try to make the name as clear as possible. Even (or especially) for foreigners.

I think the names should be mostly identical, except for something like 'single' vs. 'entire'. Maybe `hideSingleEpisodeFromViewingActivity` and `hideEntireSeriesFromViewingHistory`? I don't like how `Episode` kind of excludes movies but I couldn't come up with anything better. Do you have any ideas?

Reply:
Happy to go with `hideSingleEpisodeFromViewingActivity` and `hideEntireSeriesFromViewingHistory`. Will update code now and commit

Reply:
On second thought, will be likely next week, sorry.
If it can not wait I'm of course ok with you making these changes

Reply:
No problem at all, take your time!